db/migrate/20250903182114_add_developers.rb

```diff
@@ -0,0 +1,6 @@
+class AddDevelopers < ActiveRecord::Migration[7.2]
+  def change
+    remove_column :users, :superuser, :boolean
```

Comment:
This is a bad migration.   You will loose everyone who is already a developer.  You should utilize rename column.